app/services/mlh_table.rb

```diff
@@ -17,19 +17,19 @@ def faraday_connection
     ) do |faraday|
       faraday.use Faraday::Response::RaiseError
       faraday.adapter Faraday.default_adapter
-    end
-    response = @faraday_connection.get
-    unless Rails.configuration.cache_store == :null_store
-      response.body do
-        ActiveSupport::Cache.lookup_store(
-          *Rails.configuration.cache_store,
-          namespace: 'mlh',
-          expires_in: 3.hours
-        )
+      unless Rails.configuration.cache_store == :null_store
+        faraday.response :caching do
+          ActiveSupport::Cache.lookup_store(
+            *Rails.configuration.cache_store,
+            namespace: 'mlh',
+            expires_in: 3.hours
+          )
+        end
       end
     end
+    response = @faraday_connection.get
     return response.body if response.success?
-
+  rescue StandardError
```

Comment:
We don't want this rescue?

Reply:
So I tried testing this again, just to be sure about this behavior. If you comment out the rescue in the pages helper and add an incorrect url, you will see a completely blank screen. I've tried this twice now, rebuilding my containers both times. The rescue resolves that. But please let me know if you get different results running through that same process.